### ISSUE TITLE
Parcelize wrong error

### ISSUE THREAD

**Issue by vRallev** [uber/okbuck#774]
IntelliJ shows a wrong error when using the `@Parcelize` annotation. It doesn't understand that the Kotlin compiler will implement the `Parcelable` methods. The project compiles fine, but this error is confusing nonetheless.

I'm able to reproduce the issue with this project. After a fresh checkout I ran `./buckw project --skip-build` and opened the project in IntelliJ.

<img width="819" alt="screen shot 2018-12-18 at 12 38 11 pm" src="https://user-images.githubusercontent.com/1558045/50181725-0e663100-02c2-11e9-8393-7808c6709157.png">

Is there a workaround for this issue?

**Comment by kageiit** [comment]
I think this is related to the recent jetifier change. We may have to fix the wrapper in okbuck to generate prebuilt_jar rules like we do with aars. cc @thalescm

**Comment by thalescm** [comment]
I've switched to `v0.44.0` (before jetifier changes landed) and this still happen. So I don't think they are related.
It seems to me like this is not related to okbuck at all.
@vRallev Can you please link to a sample project where this work (message not shown) and start failing (message shown) after creating the same project with `./buckw project`?

**Comment by vRallev** [comment]
@thalescm It's possible that it never worked with okbuck. I created a new project and imported it in IntelliJ with Gradle. There I don't see the problem. With Buck I see the issue.

**Comment by thalescm** [comment]
@vRallev Have you tried a pure buck project too? (not generated by okbuck)

**Comment by vRallev** [comment]
No, I haven't. I only tried to setup the plugin in our OkBuck project.

**Comment by vRallev** [comment]
I can't rule that out especially since the Kotlin - Buck integration isn't "ideal".
